example/http/client/sync-ssl/http_client_sync_ssl.cpp

```diff
@@ -118,6 +121,13 @@ int main(int argc, char** argv)
             throw beast::system_error{ec};
 
         // If we get here then the connection is closed gracefully
+
+        // As final step we need to close underlying TCP connection
+        beast::get_lowest_layer(stream).socket().shutdown( boost::asio::socket_base::shutdown_both, ec);
```

Comment:
well this might need to be added to ALL the SSL clients :) but read this:

http://www.serverframework.com/asynchronousevents/2011/01/time-wait-and-its-design-implications-for-protocols-and-scalable-servers.html